pages/contact.html

```diff
@@ -152,7 +151,7 @@ <h6>Social</h6>
           <a href="https://l10n.opensuse.org/projects/uyuni/" target="_blank">
             <img src="../img/Weblate_logo.svg" width="24" height="24" alt="Uyuni - Weblate" title="Translate Uyuni to your language">
           </a>
-          <a href="https://gitter.im/uyuni-project" target="_blank">
+          <a href="https://www.uyuni-project.org/pages/contact.html#gitter" target="_blank">
```

Comment:
```suggestion
          <a href=".../pages/contact.html#gitter" target="_blank">
```

Reply:
Ah, now that I see this. SInce this is not an external page anymore, you can remove `target="_blank"`

```suggestion
          <a href="https://www.uyuni-project.org/pages/contact.html#gitter">
```

Same for the other places were we point to this and not directly to the channels.